libraries/Microsoft.Bot.Builder.Classic/Microsoft.Bot.Builder.Classic/packages.config

```diff
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Autofac" version="3.5.2" targetFramework="net461" />
-  <package id="Chronic.Signed" version="0.3.2" targetFramework="net461" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net461" />
-  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
-  <package id="NuGet.CommandLine" version="4.5.1" targetFramework="net461" developmentDependency="true" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Autofac" version="4.2.0" targetFramework="net461" />

```

Comment:
There is a key feature missing in the newer versions of AutoFac that were prevening us from upgrading. I don't remember what they were though. 

@tomlm, can you chime in?

Reply:
Yeah, they turned off functionality we depend on, I don't think you can do this.

---
In reply to: [181224509](https://github.com/Microsoft/botbuilder-dotnet/pull/414#discussion_r181224509) [](ancestors = 181224509)